static/app/components/events/interfaces/breadcrumbs/utils.tsx

```diff
@@ -13,7 +13,61 @@ import {
   IconWarning,
 } from 'app/icons';
 import {t} from 'app/locale';
-import {BreadcrumbType} from 'app/types/breadcrumbs';
+import {Breadcrumb, BreadcrumbLevelType, BreadcrumbType} from 'app/types/breadcrumbs';
+import {defined} from 'app/utils';
+
+function convertCrumbType(breadcrumb: Breadcrumb): Breadcrumb {
+  if (breadcrumb.type === BreadcrumbType.EXCEPTION) {
+    return {
+      ...breadcrumb,
+      type: BreadcrumbType.ERROR,
+    };
+  }
+  // special case for 'ui.' and `sentry.` category breadcrumbs
+  // TODO: find a better way to customize UI around non-schema data
```

Comment:
Could you have a map of categories, and a fallback? It looks like the `sentry.transaction` and `sentry.event` keys could also be categories in the mapping.

Reply:
hm, I'll think about it a little more and maybe update this logic in a follow-up PR ...  in this PR this code was just moved... I did not change anything